hie-plugin-api/Haskell/Ide/Engine/Compat.hs

```diff
@@ -37,79 +37,68 @@ isExtensionOf ext         = isSuffixOf ('.':ext) . takeExtensions
 #endif
 
 
-#if MIN_VERSION_ghc(8, 4, 0)
 type GhcTc = GHC.GhcTc
-#else
-type GhcTc = GHC.Id
-#endif
 
 pattern HsOverLitType :: Type.Type -> GHC.HsExpr GhcTc
 pattern HsOverLitType t <-
 #if MIN_VERSION_ghc(8, 6, 0)
     GHC.HsOverLit _ (GHC.overLitType -> t)
-#elif MIN_VERSION_ghc(8, 4, 0)
-    GHC.HsOverLit (GHC.overLitType -> t)
 #else
+    -- elif MIN_VERSION_ghc(8, 4, 0)
```

Comment:
This comment should probably come out

Reply:
I thought, it may be helpful to have them, but you are right, it is unnecessary.